LPFilter/Source/PluginEditor.cpp

```diff
@@ -57,16 +75,32 @@ void LpfilterAudioProcessorEditor::resized()
 {
     // This is generally where you'll want to lay out the positions of any
     // subcomponents in your editor..
-    bypassButton.setSize(80, 40);   // Sets button's size
+    
+    auto boundsToDivide = getLocalBounds().reduced(20);
+    auto newBounds = boundsToDivide;
+    boundsToDivide.removeFromRight(20);
+    boundsToDivide.removeFromLeft(100);
+    filterModeBox.setBounds(boundsToDivide.removeFromTop(25)); // Removes strip, reduces the rectangle and returns the strip
+    
+    boundsToDivide.removeFromTop(5);
+    
+    auto buttonBounds = newBounds.removeFromTop(30);
+    bypassButton.setSize(80, buttonBounds.getHeight());   // Sets button's size
+    bypassButton.setCentrePosition(newBounds.removeFromTop(50).getCentre());
     
 }
 
 void LpfilterAudioProcessorEditor::updateComponents()
 {
     // Update components during automation
     const bool newButtonValue = *processor.bypassParam;
+    auto newFilterType = processor.mode->getIndex();
+    
     if (newButtonValue != bypassButton.getToggleState())
         bypassButton.setToggleState( newButtonValue, dontSendNotification);
+    
+    if (newFilterType != filterModeBox.getSelectedItemIndex())
+        filterModeBox.setSelectedItemIndex(newFilterType);
```

Comment:
The notification type should be `dontSendNotification`. Otherwise, this call will trigger the `comboBoxChanged` method.